Update usability support for RHEL7

agmt opened the pull request:
Added dist's options: sslkey, sslcert.
INFO: https://github.com/dagwieers/mrepo/issues/71

agmt committed:
:

agmt committed:
f

agmt committed:
Config

elentar committed:
Support for CA cert

SeredinYegor committed:
Merge pull request #1 from elentar/ca-support

Support for CA certs

ghoneycutt commented:
Hi @dagwieers 

Are you still maintaining this?

AbhishekKataria commented:
I see no official centos7 release rpm for mrepo. So it is safe to use mrepo-el6 for the same? 
Where can I get one for el7? Is there any alternatives I can use?

wernermaes commented:
Hello

I think it’s no problem to use mrepo-el6 on centos7.
I don’t know whether  there is still development being done on mrepo

Kind regards

Werner Maes

Van: Abhishek Kataria [mailto:notifications@github.com]
Verzonden: dinsdag 24 januari 2017 3:31
Aan: dagwieers/mrepo <mrepo@noreply.github.com>
CC: wernermaes <werner@kuleuven.be>; Comment <comment@noreply.github.com>
Onderwerp: Re: [dagwieers/mrepo] Update usability support for RHEL7 (#75)


I see no official centos7 release rpm for mrepo. So it is safe to use mrepo-el6 for the same?
Where can I get one for el7? Is there any alternatives I can use?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/dagwieers/mrepo/pull/75#issuecomment-274683633>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AILhnd0PJvrv-FYPvSEvhWBlkI4gDS-qks5rVWJcgaJpZM4CULDS>.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/dagwieers/mrepo","title":"dagwieers/mrepo","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/dagwieers/mrepo"}},"updates":{"snippets":[{"icon":"PERSON","message":"@AbhishekKataria in #75: I see no official centos7 release rpm for mrepo. So it is safe to use mrepo-el6 for the same? \r\nWhere can I get one for el7? Is there any alternatives I can use?"}],"action":{"name":"View Pull Request","url":"https://github.com/dagwieers/mrepo/pull/75#issuecomment-274683633"}}}

AbhishekKataria commented:
Hi Werner, Thanks for your answer. I appreciate it. 
Abhishek

derynimind commented:
I used the software on rhel7 almost 2 years ago, it worked well. I had to
compile and build an install from source. If you get stuck I still have my
notes.

On Jan 24, 2017 12:09 PM, "Abhishek Kataria" <notifications@github.com>
wrote:

> Hi Werner, Thanks for your answer. I appreciate it.
> Abhishek
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/dagwieers/mrepo/pull/75#issuecomment-274886755>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AJmYmubr3fyseJSw4x7xsSqbPv_8-Xd5ks5rVj5mgaJpZM4CULDS>
> .
>

campfield committed:
Add support for IBM ppc64le architecture.

IBM Power line of processors now includes the architecture ppc46le
with matching architecture identifiers in repository URLs.  Add
value of ppc64le to archs so mrepo will view this value as a valid
architecture when used in configuration files.

SeredinYegor committed:
Merge pull request #2 from campfield/feature/support_ppc64le_architecture

Add support for IBM ppc64le architecture.

agmt committed:
Merge branch 'dagwieers:master' into master
